src/emulator/functionsEmulator.ts

```diff
@@ -833,9 +839,14 @@ export class FunctionsEmulator implements EmulatorInstance {
       .filter((t) => !t.backend.extensionInstanceId)
       .map((t) => t.def);
     return this.args.emulatableBackends.map((e: EmulatableBackend) => {
+      const envWithSecrets = Object.assign({}, e.env);
```

Comment:
Don't we typically use the splat operator now instead of `Object.assign`? Silly nit but could just have said `{ ...e.env }`

Reply:
Ack - I'll slip this into a future PR since this already went in.